e2e-tests/gef/cypress/integration/unissued-facilities-AIN.spec.js

```diff
@@ -0,0 +1,523 @@
+      aboutFacilityUnissued.errorSummary().contains('The issue date must be within 3 months of the inclusion notice submission date');
+      aboutFacilityUnissued.issueDateDay().clear();
+      aboutFacilityUnissued.issueDateMonth().clear();
+      aboutFacilityUnissued.issueDateYear().clear();
+      aboutFacilityUnissued.issueDateDay().type(oneMonthDay);
+      aboutFacilityUnissued.issueDateMonth().type(oneMonthMonth);
+      aboutFacilityUnissued.issueDateYear().type(oneMonthYear);
+      aboutFacilityUnissued.shouldCoverStartOnSubmissionNo().click();
+      aboutFacilityUnissued.coverStartDateDay().type(twentyEightDay);
+      aboutFacilityUnissued.coverStartDateMonth().type(twentyEightMonth);
+      aboutFacilityUnissued.coverStartDateYear().type(twentyEightYear);
+      aboutFacilityUnissued.continueButton().click();
+      aboutFacilityUnissued.coverStartDateError().contains('Cover start date cannot be before the issue date');
+      aboutFacilityUnissued.errorSummary().contains('Cover start date cannot be before the issue date');
+      aboutFacilityUnissued.coverStartDateDay().clear();
+      aboutFacilityUnissued.coverStartDateMonth().clear();
+      aboutFacilityUnissued.coverStartDateYear().clear();
+      aboutFacilityUnissued.coverStartDateDay().type(threeMonthsOneDayDay);
+      aboutFacilityUnissued.coverStartDateMonth().type(threeMonthsOneDayMonth);
+      aboutFacilityUnissued.coverStartDateYear().type(threeMonthsOneDayYear);
+      aboutFacilityUnissued.continueButton().click();
+      aboutFacilityUnissued.coverStartDateError().contains('The cover start date must be within 3 months of the inclusion notice submission date');
+      aboutFacilityUnissued.errorSummary().contains('The cover start date must be within 3 months of the inclusion notice submission date');
+      aboutFacilityUnissued.coverStartDateDay().clear();
+      aboutFacilityUnissued.coverStartDateMonth().clear();
+      aboutFacilityUnissued.coverStartDateYear().clear();
+      aboutFacilityUnissued.coverStartDateDay().type(twoMonthsDay);
+      aboutFacilityUnissued.coverStartDateMonth().type(twoMonthsMonth);
+      aboutFacilityUnissued.coverStartDateYear().type(twoMonthsYear);
+      aboutFacilityUnissued.coverEndDateDay().type(twentyEightDay);
+      aboutFacilityUnissued.coverEndDateMonth().type(twentyEightMonth);
+      aboutFacilityUnissued.coverEndDateYear().type(twentyEightYear);
+      aboutFacilityUnissued.continueButton().click();
+      aboutFacilityUnissued.coverEndDateError().contains('Cover end date cannot be before the issue date');
+      aboutFacilityUnissued.errorSummary().contains('Cover end date cannot be before cover start date');
+      aboutFacilityUnissued.errorSummary().contains('Cover end date cannot be before the issue date');
+    });
+
+    it('the correct success messages should be displayed', () => {
```

Comment:
Can we be more descriptive? 'success messages should be display after .......' ?

Reply:
done